python_streamer/bluesky/create_post.py

```
@@ -8,9 +10,9 @@
 
 @final
 class CreatePostStreamer(AbstractStreamerWebsocket[CommitWrapperEventModel]):
-    # N_CACHED = 200_000
-    # cached_models: List[CommitWrapperEventModel] = []
-    # pbar: tqdm = tqdm(total=N_CACHED)
+    N_CACHED = 20_000
+    cached_models: List[CommitWrapperEventModel] = []
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)